modules/azure/aks.go

```diff
@@ -0,0 +1,72 @@
+package azure
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2019-11-01/containerservice"
+	"github.com/gruntwork-io/terratest/modules/k8s"
+	"github.com/gruntwork-io/terratest/modules/logger"
+	"github.com/gruntwork-io/terratest/modules/retry"
+)
+
+// GetManagedClustersClient is a helper function that will setup an Azure ManagedClusters client on your behalf
+func GetManagedClustersClient(subscriptionID string) (*containerservice.ManagedClustersClient, error) {
```

Comment:
Terratest convention is to use `E` to denote functions that return an `error`. Functions without `E` should use `require` to fail the test if there is an error. This applies to all functions in this file.

Reply:
Done